Jenkinsfile-core-dev

```diff
@@ -0,0 +1,59 @@
+pipeline {
+    agent {
+        label 'ec2-jdk8'
+    }
+
+    triggers {
+        pollSCM('H/30 * * * *')
+    }
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '5'))
+    }
+
+    environment {
+        MAVEN_OPTS = '-Xms1024m -Xmx4096m'
+    }
+
+    stages {
+        stage ('Build') {
+            steps {
+                echo 'Building DHIS2 ...'
+                script {
+                    withMaven {
+                        sh 'mvn -T 4 clean install -f dhis-2/pom-full.xml -Pjdk8 --update-snapshots'
+                    }
+                }
+            }
+        }
+
+        stage ('Sync WAR') {
+            steps {
+                echo 'Syncing WAR ...'
+                sh 'curl "https://raw.githubusercontent.com/dhis2/dhis2-server-setup/master/ci/scripts/sync-war-s3.sh" -o sync-war-s3.sh'
+                sh 'chmod +x sync-war-s3.sh'
+                sh './sync-war-s3.sh ${GIT_BRANCH}'
+            }
+        }
+
+        stage ('Update Play Instance') {
+            steps {
+                echo 'Updating Play Instance ...'
+                sh 'curl "https://raw.githubusercontent.com/dhis2/dhis2-server-setup/master/ci/scripts/update-play-instance.sh" -o update-play-instance.sh'
+                sh 'chmod +x update-play-instance.sh'
+                script {
+                    withCredentials([usernameColonPassword(credentialsId: 'awx-bot-user-credentials', variable: 'AWX_AUTH')]) {
+                        sh './update-play-instance.sh dev $AWX_AUTH'
```

Comment:
This won't fail if ansible job fails, right? I think it would be nice for visibility, but not required.

Reply:
That's correct - it won't know if the actual ansible template run successfully or not. Given that we're moving to Kubernetes as a means of managing the instances in the future - we could have that type of feedback there.